ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -1353,8 +1383,8 @@ generateGhcLibCabal ghcFlavor customCppOpts = do
             (Set.fromList parserModules))
       ```
   -}
-  let hsSrcDirs = replace ["libraries/ghc-boot-th/../ghc-internal/src"] [] (ghcLibHsSrcDirs False ghcFlavor lib)
-  let includeDirs = replace ["libraries/ghc-internal/include"] [] (ghcLibIncludeDirs ghcFlavor)
+  let hsSrcDirs = ghcLibHsSrcDirs False ghcFlavor lib
+  let includeDirs = ghcLibIncludeDirs False ghcFlavor
```

Comment:
Same as above?

Reply:
in this case we pass through `forDepends` in a context in which it is `False`. in `ghcLibIncludeDirs` it is not currently inspected but that's ok. it's good to have the capability there.